manifests/init.pp

```diff
@@ -107,8 +106,14 @@
   $stop_jira = 'service jira stop && sleep 15',
 
   # Tomcat
-  $tomcatAddress = undef,
-  $tomcatPort = 8080,
+  $tomcatAddress      = undef,
+  $tomcatPort         = 8080,
+  $tomcatNativeSsl    = false,
+  $tomcatHttpsPort    = 8443,
+  $tomcatKeyAlias     = 'jira',
+  $tomcatKeystoreFile = '/home/jira/jira.jks',
+  $tomcatKeystorePass = 'changeit',
+  $tomcatKeystoreType = 'JKS',
```

Comment:
I Agree it needs to follow the current puppetlabs style guide, but this needs to happen as part of a major version upgrade.

I did not want to start now, and then its mixed between camelcase and underscores.